docs/en/index.rst

```diff
@@ -279,7 +279,7 @@ by using the `tableize`_ method and replacing underscores with hyphens:
 
     public static function slugify(string $text) : string
     {
-        return str_replace('_', '-', Inflector::tableize($text));
+        return str_replace('_', '-', self::$inflector->tableize($text));
```

Comment:
Could you instead follow the examples above? So get rid of entire static function thing and leave `echo str_replace('_', '-', $inflector->tableize($text));`? It will be consistent and will prevent questions how to get `self::$inflector` :)